controllers/userController.js

```diff
@@ -0,0 +1,25 @@
+const userModel = require("../models/user");
+const roles = require("../models/roles")
+const bcrypt = require('bcrypt');
+const userRegister = async (req, res) => {
+  try {
+    const { username, password, confirmPassword, email, firstName, lastName, roleId } = req.body
+
+    if (password == confirmPassword) {
+      const searchIdInRoles = await roles.findOne({ where: { id: roleId } })
```

Comment:
you created a function in role controller to get role by id, use that function to get role from db @ashok-choudhary-0